Gemfile.lock

```diff
@@ -345,8 +345,8 @@ DEPENDENCIES
   jquery-turbolinks
   jquery-ui-rails
   less-rails
-  owlcarousel-rails!
   mini_magick
+  owlcarousel-rails!
```

Comment:
since this is supposed to be alphabetical some prior pr must have been wrong on this